test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -1083,3 +1159,13 @@ func busyCpuImageEnv() string {
 	}
 	return fmt.Sprintf("%s%s", qeImageRegistry, busyCpusImage)
 }
+
+// isPodReady checks if the pod is in ready state
+func isPodReady(pod *corev1.Pod) bool {
+	for _, condition := range pod.Status.Conditions {
+		if condition.Type == corev1.PodReady && condition.Status == corev1.ConditionTrue {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
there is a util function to check that https://github.com/openshift/cluster-node-tuning-operator/blob/ea36e6ae28ce38898481d5f4cd8819a6e017c95b/test/e2e/util/wait/pod.go#L7